### PR TITLE
Feature/UI kit 3043/integrate voice message

### PR DESCRIPTION
**Pull request by HoonBaek**
## For Internal Contributors

[UIKIT-3043](https://sendbird.atlassian.net/browse/UIKIT-3043)

## Description Of Changes

Add a brief description of the changes that you have involved in this PR

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [ ] Bugfix
- [x] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [ ] Improvement (refactor code)

[UIKIT-3043]: https://sendbird.atlassian.net/browse/UIKIT-3043?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/hooks/useVoiceRecorder/index.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const VoiceRecorderProvider = (props: VoiceRecorderProps): React.ReactEle
         const mediaRecorder = new MediaRecorder(stream);
         mediaRecorder.ondataavailable = (e) => {// when recording stops
           // Generate audio file
-          const audioFile = new File([e.data], 'I am file name', {
+          const audioFile = new File([e.data], 'Voice message', {
```
Comment (sravan-s) [inline src/hooks/useVoiceRecorder/index.tsx:43]: Voice message need extension, no?

File: src/lib/Sendbird.jsx [hunk 2]
```diff
@@ -262,6 +272,11 @@ Sendbird.propTypes = {
   colorSet: PropTypes.objectOf(PropTypes.string),
   isReactionEnabled: PropTypes.bool,
   isMentionEnabled: PropTypes.bool,
+  isVoiceMessageEnabled: PropTypes.bool,
+  voiceRecord: PropTypes.shape({
+    maxRecordingTime: PropTypes.number,
+    minRecordingTime: PropTypes.number,
+  }),
```
Comment (sravan-s) [inline src/lib/Sendbird.jsx:279]: discuss: isnt it better for this prop to be in Channel?

File: src/smart-components/Channel/components/MessageInput/voice-message-wrapper.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@import '../../../../styles/variables';
+
+.sendbird-voice-message-input-wrapper .sendbird-voice-message-input__indicator {
+  height: 40px;
+}
```
Comment (sravan-s) [inline src/smart-components/Channel/components/MessageInput/voice-message-wrapper.scss:5]: Isnt `.sendbird-voice-message-input__indicator` enough?

File: src/smart-components/Thread/components/ThreadList/ThreadListItemContent.tsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ import UserProfile from '../../../../ui/UserProfile';
 import MessageItemMenu from '../../../../ui/MessageItemMenu';
 import MessageItemReactionMenu from '../../../../ui/MessageItemReactionMenu';
 import Label, { LabelTypography, LabelColors } from '../../../../ui/Label';
-import { getClassName, getSenderName, getUIKitMessageType, getUIKitMessageTypes, isOGMessage, isTextMessage, isThumbnailMessage } from '../../../../utils';
+import { getClassName, getSenderName, getUIKitMessageType, getUIKitMessageTypes, isOGMessage, isTextMessage, isThumbnailMessage, isVoiceMessage } from '../../../../utils';
```
Comment (sravan-s) [inline src/smart-components/Thread/components/ThreadList/ThreadListItemContent.tsx:17]: split into newline pls, sorry, I should have added lint ~

**Comment by HoonBaek** [comment]
https://github.com/sendbird/sendbird-uikit-react/pull/405

**Pull request closed without merging**